### DISCUSSION TITLE
Configuration not valid

### DISCUSSION THREAD

**Discussion by mickeheterjag** [djansen1987/SAJeSolar#62] [Q&A] [answered]
I am completely new to this so I apologize in advance if the question is very simple.

I have problems with the configuration itself as I get an error message that reads: Platform error sensor.saj_esolar - Exception importing custom_components.saj_esolar.sensor

I have tried to solve the problem by installing the integration both via HACS and manually. But the problem remains.

I have a SAJ R6 with Esolar SEC-C module.

Thanks in advance

**Comment by djansen1987** [comment]
Hi @mickeheterjag ,

There was an issue with the previous update. Sorry.. it should be fixed in the latest which just came out

**Reply by mickeheterjag** [reply, marked as answer]
Now it´s working. Thank you very much!

**Comment by 78lobo78** [comment]
buenas, alguien sabe que le pasan a los sensores que no funcionan. Me salen desconocidos
